cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -899,10 +899,14 @@ function Start-WorkerServices {
   $kubelet_args = $kubelet_args_str.Split(" ")
   Log-Output "kubelet_args from metadata: ${kubelet_args}"
 
-  $additional_arg_list = @(`
+  $kubeproxy_args_str = ${kube_env}['KUBEPROXY_ARGS']
+  $kubeproxy_args = $kubelet_args_str.Split(" ")
+  Log-Output "kubeproxy_args from metadata: ${kubeproxy_args}"
+
+  $default_kubelet_args = @(`
       "--pod-infra-container-image=${INFRA_CONTAINER}"
   )
-  $kubelet_args = ${kubelet_args} + ${additional_arg_list}
+  $kubelet_args = ${default_kubelet_args} + ${kubelet_args}
```

Comment:
Can you move lines 906-909 above line 902, so that all the kubelet arg stuff happens first, then all the kubeproxy arg stuff?

Reply:
Done.